src/test/kotlin/calculator/CalculatorTest.kt

```diff
@@ -0,0 +1,84 @@
+package calculator
+
+import org.assertj.core.api.Assertions.assertThat
+import org.assertj.core.api.Assertions.assertThatExceptionOfType
+import org.junit.jupiter.api.Test
+import org.junit.jupiter.params.ParameterizedTest
+import org.junit.jupiter.params.provider.ValueSource
+
+class CalculatorTest {
+    @Test
+    fun `더하기만 테스트`() {
+        val stringCalculator1 = StringCalculator("1 + 3")
+        val result1 = stringCalculator1.result
+        assertThat(result1).isEqualTo((1 + 3).toDouble())
+
+        val stringCalculator2 = StringCalculator("15 + 31")
+        val result2 = stringCalculator2.result
+        assertThat(result2).isEqualTo((15 + 31).toDouble())
+    }
```

Comment:
```suggestion
    @ParameterizedTest
    @CsvSource(value = ["1 + 3,4", "15 + 31,46"])
    fun `더하기만 테스트`(input: String, expect: Double) {
        val stringCalculator = StringCalculator(input)
        val result = stringCalculator.result
        assertThat(result).isEqualTo(expect)
    }
```
`@CsvSource` 를 사용하여 테스트하면, 여러 케이스에 대해 테스트를 하실 수 있습니다 😊